pydantic/config.py

```diff
@@ -563,7 +563,7 @@ class Transaction(BaseModel):
     3. Using `'never'` we would have gotten `user=SubUser(hobbies=['scuba diving'], sins=['lying'])`.
     """
 
-    ser_json_timedelta: Literal['iso8601', 'float']
+    ser_json_timedelta: Literal['iso8601', 'float', 'millisecond']
     """
     The format of JSON serialized timedeltas. Accepts the string values of `'iso8601'` and
```

Comment:
Will need to change the docstring here.

I'm wondering if we should add support for `'second'` as well, and deprecate `'float'` down the line (equivalent to `'second'`...